src/components/Form/Form.tsx

```diff
@@ -0,0 +1,124 @@
+import "./style.scss";
+import { useFormik } from "formik";
+import { Film, Modal } from "../../interfaces";
+import { films } from "../../films";
+import Select from "react-select";
+import DeleteForm from "./DeleteForm";
+import { useEffect, useState } from "react";
+import { newMovie, FormType, Genres } from "../../constants";
+import { selectStyle } from "./selectConfig";
+
+interface FormProps {
+  closeModal(): void;
+  modalState: Modal;
+}
+
+const Form: React.FC<FormProps> = ({ closeModal, modalState }) => {
+  const [movie] = useState<Film>(
+    films.find((el) => {
+      return el.id === modalState.filmID;
+    }) || newMovie
+  );
+
+  useEffect(() => {
+    document.querySelector("body")?.classList.add("overflow-hidden");
+    return () => {
+      document.querySelector("body")?.classList.remove("overflow-hidden");
+    };
+  });
```

Comment:
this effect would be run on each rerender.
And this logic should be outside form, i think it should be within some ModalWrapper level. What if tomorrow you need another modal to handle.